docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md

```diff
@@ -31,21 +31,25 @@ chmod +x sgx_linux_x64_driver.bin
 sudo ./sgx_linux_x64_driver.bin
 ```
 
-Note: This may not be the latest Intel SGX DCAP driver. Please check with [Intel's SGX site](https://01.org/intel-software-guard-extensions/downloads) if a more recent SGX DCAP driver exists.
+> This may not be the latest Intel SGX DCAP driver.
+> Please check with [Intel's SGX
```

Comment:
The rest of this already exceeds the 80 char limit, so might as well let the line run long for readability as in the 18.04 version of this doc.

Reply:
Oops fixed thanks